### PR TITLE
Organize test on TravisCI

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

**Comment by tagomoris** [comment]
Currently, I'm not willing to support Fluentd v0.10 with this plugin. It doesn't mean that I want to break compatibility for v0.10, but it's too much to make sure to be green for v0.10 ever after.

**Comment by okkez** [comment]
> Currently, I'm not willing to support Fluentd v0.10 with this plugin. It doesn't mean that I want to break compatibility for v0.10, but it's too much to make sure to be green for v0.10 ever after.

I got it. I will remove CI configuration for Fluentd v0.10.

File: fluent-plugin-webhdfs.gemspec [hunk 1]
```diff
@@ -17,7 +17,8 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency "rake"
   gem.add_development_dependency "test-unit"
-  gem.add_runtime_dependency "fluentd", '>= 0.10.53'
+  gem.add_development_dependency "appraisal"
+  gem.add_runtime_dependency "fluentd", '>= 0.10.59'
```
Comment (tagomoris) [inline fluent-plugin-webhdfs.gemspec]: Why this upgrade should be done? Are there any incompatibilities with 0.10.53 - 0.10.58?

Comment (okkez) [inline fluent-plugin-webhdfs.gemspec]: Because `@type` notation is since Fluentd v0.10.59.
We use `@type` notation In README.md.

Comment (tagomoris) [inline fluent-plugin-webhdfs.gemspec]: Ah, got it. Thanks.

**Pull request merged**